packages/babel-plugin-proposal-decorators/src/transformer-2022-03.ts

```diff
@@ -101,12 +102,15 @@ function createLazyPrivateUidGeneratorForClass(
  */
 function replaceClassWithVar(
   path: NodePath<t.ClassDeclaration | t.ClassExpression>,
+  className: string | t.Identifier | t.StringLiteral | undefined,
 ): [t.Identifier, NodePath<t.ClassDeclaration | t.ClassExpression>] {
   if (path.type === "ClassDeclaration") {
+    const id = path.node.id;
+    const className = id.name;
```

Comment:
```suggestion
    const className =  path.node.id.name;
```
It seems that `id` is only used once?